src/main/java/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/ExplosivePickaxe.java

```diff
@@ -77,7 +80,7 @@ else if (b.getType().name().endsWith("_SHULKER_BOX")) {
 									b.breakNaturally();
 								}
 								else {
-									for (ItemStack drop: b.getDrops()) {
+									for (ItemStack drop: b.getDrops(EFFECTIVE_PICKAXE)) {
```

Comment:
Please add a space before the :
to follow our new conventions.

(Would be awesome if you could change that in all the for loops of the other files you edited too)